UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -59,6 +61,8 @@ public class CargoManager : MonoBehaviour
 
 		private static readonly List<int> randomJunkPrices = new List<int> { 5, 10, 15 };
 
+		public static List<string> researchedArtifacts = new List<string>();
```

Comment:
This needs to be cleared on round end and add this to the file script somewhere:

```
[RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
		private static void ClearStatics()
		{
			researchedArtifacts = new List<string>();
		}
```